### PR TITLE
Add real-world data to training data

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

File: base/data/nlu/address.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+- Vall
+- Haddon
+- Tigane
+- Lochnerhof
+- Moloto block 20
+- Soweto naledi
+- Toti
+- Zola2
+- Klerksdorp,
+jouberton -
```
Comment (miltontony) [inline base/data/nlu/address.md:70]: line break

File: base/data/nlu/address.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+- Matz
+- New eeasterus
+- China gate
+- Tarlton
+- Mutititi
+- Virginia Freestate South Africa
+- Somersetwest
+- iNANDA DAM
+- Meerensee
+- (+
```
Comment (miltontony) [inline base/data/nlu/address.md:234]: These need to be filtered out

File: base/data/nlu/address.md (hunk 1 above)
Comment (rudigiesler) [reply base/data/nlu/address.md:70]: This is a multi-line message from a user. I didn't see any errors/warnings during training, so I assumed that this was the correct way of representing it in the data.

File: base/data/nlu/address.md (hunk 2 above)
Comment (rudigiesler) [reply base/data/nlu/address.md:234]: This was from the text of the location question, so I assumed that it was related. I would rather have these classified as "location", and then us returning an error asking them to try again, than it being accidentally classified as an "exit" message

Comment (rudigiesler) [reply base/data/nlu/address.md:234]: The other alternative is to classify these as "chitchat", which we could also do.

**Pull request merged**